fpga_interchange/testarch_generators/generate_testarch.py

```diff
@@ -79,12 +79,12 @@ def make_slice_site_type(self):
         w = site_type.add_wire("LUT_O")
         w.connect_to_bel_pin("LUT", "O")
         w.connect_to_bel_pin("FFMUX", "I0")
+        w.connect_to_bel_pin("O","O")
 
         w = site_type.add_wire("MUX_O")
         w.connect_to_bel_pin("FFMUX", "O")
         w.connect_to_bel_pin("FF", "D")
 
-        w = site_type.add_wire("LUT_OUT", [("LUT", "O"), ("O", "O")])
```

Comment:
I had to remove wire "LUT_OUT" as there cannot be 2 wires driven by the same output.
This caused nextpnr_emit to fail

Reply:
Yeah, this looks like a typo in any case as the `O` site pin is being correctly driven by the `LUT_O` wire anyway: https://github.com/antmicro/python-fpga-interchange/blob/399355e63c3edf9202ca373d371cfa7d1bfa92a4/fpga_interchange/testarch_generators/generate_testarch.py#L82